### ISSUE TITLE
dnsmasq for multiple interface does not work with debian

### ISSUE THREAD

**Issue by do3cc** [debops/ansible-dnsmasq#7]
I have the same configuration for dnsmasq on debian and mint (ubuntu)
On debian, dnsmasq fails with the error message that the interface br2 does not exist.
Running dnsmasq with only the config file for interface_br2, makes it work. Then I cannot start dnsmasq with the other config file...
debian has dnsmasq 2.62, ubunt 2.68
The changelog mentions in 2.63, that the feature of declaring an interface for tftp-enable has been removed, because of weird syntax and no users.
In 2.67 the feature has been added based on someones suggestions.
I think that maybe  the implementation has changed and this is the reason why it works in mint.
No idea how to solve this, though.

**Comment by drybjed** [comment]
Are you trying this on Jessie? The way to handle this correctly would be to check the available version of dnsmasq and surround the options with `{% version_compare() %}` sections to only enable relevant options if a particular version is defined. `dnsmasq_version` variable contains the current version available.

**Comment by do3cc** [comment]
Debian wheezy, 7.7
Problem is, I find no configuration that would allow me dnsmasq with different configs and tftp on br2

On 13. Januar 2015 08:59:07 MEZ, Maciej Delmanowski <notifications@github.com> wrote:
>Are you trying this on Jessie? The way to handle this correctly would
>be to check the available version of dnsmasq and surround the options
>with `{% version_compare() %}` sections to only enable relevant options
>if a particular version is defined. `dnsmasq_version` variable contains
>the current version available.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/debops/ansible-dnsmasq/issues/7#issuecomment-69708026

-- 
Diese Nachricht wurde von meinem Mobiltelefon mit Kaiten Mail gesendet.

**Comment by drybjed** [comment]
TFTP support in dnsmasq is enabled by default on `br2` when you run the role. I'm using it to provision new KVM servers all the time on the internal network. Maybe we could arrange a screen sharing sesstion so I could see your configuration later in the day?

**Comment by do3cc** [comment]
Yes, I'll be in the office in half an hour. Let's tell me on irc when is a good time. Meanwhile I try to add my config to this ticket

On 13. Januar 2015 09:14:12 MEZ, Maciej Delmanowski <notifications@github.com> wrote:
>TFTP support in dnsmasq is enabled by default on `br2` when you run the
>role. I'm using it to provision new KVM servers all the time on the
>internal network. Maybe we could arrange a screen sharing sesstion so I
>could see your configuration later in the day?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/debops/ansible-dnsmasq/issues/7#issuecomment-69709276

-- 
Diese Nachricht wurde von meinem Mobiltelefon mit Kaiten Mail gesendet.

**Comment by ypid** [comment]
@do3cc This issue is abit old. Where you able to resolve it and if yes, could you close :wink: ?

**Comment by do3cc** [comment]
Looking at https://packages.debian.org/search?keywords=dnsmasq this might be working now, but I don't have anything to test or reproduce it.
